scripts/generate_weights.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/sh
```

Comment:
This generates weights only for the messages pallet; I don't think you meant to check this in.  Either way it should be removed.